website/docs/reference/warehouse-setups/dremio-setup.md

```diff
@@ -5,7 +5,7 @@ meta:
   authors: 'Dremio (formerly Fabrice Etanchaud)'
   github_repo: 'dremio/dbt-dremio'
   pypi_package: 'dbt-dremio'
-  min_core_version: 'v1.1.0'
+  min_core_version: 'v1.3.0'
   cloud_support: Not Supported
```

Comment:
This is referring to dbt Cloud? If so you are correct - not supported

Reply:
Yes, that refers to dbt Cloud. Thanks.